Lib/asyncio/tasks.py

```diff
@@ -382,7 +382,11 @@ def create_task(coro, *, name=None):
         raise ValueError(f'Invalid return_when value: {return_when}')
 
     if loop is None:
-        loop = events.get_event_loop()
+        loop = events.get_running_loop()
+    else:
+        warnings.warn("The loop argument is deprecated and scheduled for"
```

Comment:
All of the warning messages are missing spaces (between "for" and "removal").

Reply:
You're right. @joaojunior please submit another PR to fix this.

@willingc I don't think @asvetlov had a chance to review this PR :(

Reply:
@1st1 Ok, I will send.

Reply:
Oops. Sorry @1st and @asvetlov. I messed up on that. 😞

Reply:
@joaojunior never mind, it's handled in https://github.com/python/cpython/pull/9579